packages/react-components/react-progress/src/components/Progress/Progress.tsx

```diff
@@ -6,7 +6,7 @@ import type { ProgressProps } from './Progress.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
 
 /**
- * Progress component - TODO: add more docs
+ * Converged Progress component for the fluentui repo
```

Comment:
```suggestion
 * Converged Progress component for v9's Fluent UI library
```

Or maybe adding a description of the component would be preferable?
https://github.com/microsoft/fluentui/blob/b9e6c6a7dd8ce6111744739a0ec486b99c2bb2ea/packages/react-components/react-spinbutton/src/components/SpinButton/SpinButton.tsx#L9